draft-ietf-quic-invariants.md

```diff
@@ -189,7 +189,7 @@ version-specific semantics are marked with an X.
 +-+-+-+-+-+-+-+-+
 |0|X X X X X X X|
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-|                Destination Connection ID (0..144)           ...
+|                 Destination Connection ID (*)               ...
```

Comment:
What about DCIL and long headers?

Reply:
This is the short header only.  See the section header.

Reply:
yes, but why are short header CID's arbitrary when long headers are not? Isn't the short header CID length the same as those in the long header?

Reply:
OK, I see - invariants, not transport - and negotatied CIDs can change length, just not in long headers.